tests/framework/utils/k8s_helper.go

```diff
@@ -87,7 +95,12 @@ func CreateK8sHelper(t func() *testing.T) (*K8sHelper, error) {
 	return h, err
 }
 
-var k8slogger = capnslog.NewPackageLogger("github.com/rook/rook", "utils")
+var (
+	k8slogger = capnslog.NewPackageLogger("github.com/rook/rook", "utils")
+	CliCmd    = GetCmd()
```

Comment:
Does these variables need to be exported?

Reply:
For `cliCmd` we don't need to export but for ` RetryLoop` yes.